frontend/src/routes/person/[id].svelte

```diff
@@ -11,9 +11,11 @@
     const PERSON_DETAIL_URL: string = `${variables.apiPath}/person/${$page.params.id}`;
     const IMG_URL: string = 'https://image.tmdb.org/t/p/original/';
     const LOW_RES_IMG_URL: string = 'https://image.tmdb.org/t/p/w500/';
+    const INITIAL_BIOGRAPHY_LENGTH: number = 200;

```

Comment:
I would probably put in a larger number until we have more things to show, it feels a little empty(around ~500-ish)

Reply:
Put on 500 :D